### PR TITLE
Zoho Books updates

### PR DESCRIPTION
**Pull request by jepmatthews**
# Description

Fix some links, fix table and updated some text in the rate limit section 

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [x] Fixes
- [x] Styling

### CODE REVIEW

File: docs/integrations/accounting/zoho-books/zoho-book-limits.md [hunk 1]
```diff
@@ -5,38 +5,18 @@ createdAt: "2020-04-06T07:37:50.107Z"
...
-- `taxAmount`",
-  "3-0": "Invoices",
-  "3-1": "Line items"
-  },
-  "cols": 2,
-  "rows": 4
-  }
-  
-  To remove these restrictions and pull complete datasets for the supported Zoho Books data types, please contact [Codat Support](mailto:support@codat.io). **After removing the restrictions, fetching large volumes of data from the above data types can take multiple days to complete.**
+| Data type pulled  | Fields excluded                 |
```
Comment (mcclowes) [inline docs/integrations/accounting/zoho-books/zoho-book-limits.md:12]: 🙌

**Review by mcclowes — APPROVED** [review]
Thanks for these changes!

File: docs/integrations/accounting/zoho-books/zoho-book-limits.md [hunk 2]
```diff
@@ -51,22 +31,22 @@ The Zoho Books API has the following connection limits and rate limits:
 - Per-minute rate limit
 - Refresh token limit
 
-Our Zoho Books integration works within these limits to minimize the performance impact of pulling and pushing large volumes of data.
+Our Zoho Books integration works within these limits to minimize the performance impact of pulling and pushing large volumes of data. It tries to maximise the number of records retrieved per API call, but for some data types or if the retrictions described above are removed it is only possible to fetch a single record for each API call.
```
Comment (d-laing) [inline docs/integrations/accounting/zoho-books/zoho-book-limits.md:34]:
```suggestion
Our Zoho Books integration works within these limits to minimize the performance impact of pulling and pushing large volumes of data. To do so, the integration maximises the number of records retrieved per API call for most data types. However, for certain data types, the integration can only fetch a single record for each API call to Zoho Books, impacting performance. This also applies if the restrictions on pulling reduced datasets described above are removed.
```

File: docs/integrations/accounting/zoho-books/zoho-book-limits.md [hunk 3]
```diff
@@ -51,22 +31,22 @@ The Zoho Books API has the following connection limits and rate limits:
...
 
 ### API daily rate limit
 
-The _API daily rate limit_ lets organizations retrieve up to 2,500 data records from Zoho Books per day; for example, invoices, bills, and payments.
+The _API daily rate limit_ lets organizations make a limited number of API calls to Zoho Books per day (up to 2,500 or 500 per user whichever is lower);
 
-If the cumulative number of data records synchronized across your linked companies exceeds the daily limit, you might encounter the following issues:
+If the cumulative number of API calls to the organisation you are trying to link to exceeds the daily limit, you might encounter the following issues:
```
Comment (d-laing) [inline docs/integrations/accounting/zoho-books/zoho-book-limits.md:40]:
```suggestion
If the cumulative number of API calls to the organization you are trying to link to exceeds the daily limit, you might encounter the following issues:
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Thanks. I made a few suggestions.

File: docs/integrations/accounting/zoho-books/zoho-book-limits.md (hunk 2 above)
Comment (d-laing) [reply docs/integrations/accounting/zoho-books/zoho-book-limits.md:34]: I have tried to break up the information a bit.

File: docs/integrations/accounting/zoho-books/accounting-zoho-books.md [hunk 4]
```diff
@@ -11,11 +11,11 @@ You can synchronize accounting data with <a href="https://www.zoho.com/books/" t
...
+For more information about the supported data types, see [Zoho Books integration reference](zoho-books-integration-reference).
 
 ## Set up the integration
 
-See [Set up the Zoho Books integration](/integrations/accounting/zohobooks/accounting-zohobooks-setup) to learn how to set up and enable the integration.
+See [Set up the Zoho Books integration](accounting-zohobooks-setup) to learn how to set up and enable the integration.
 
 :::caution Reduced datasets
-By default, reduced datasets are pulled for the Bills, Credit Notes, Customers, and Invoices data types. For more information, see [Zoho Books limits and reduced data pulls](/zoho-book-limits).
+By default, reduced datasets are pulled for the Bills, Credit Notes, Customers, and Invoices data types. For more information, see [Zoho Books limits and reduced data pulls](zoho-book-limits).
```
[the remark this answers is not in the document]
Comment (jepmatthews) [reply docs/integrations/accounting/zoho-books/accounting-zoho-books.md:21]: Happy to do this, but if you move the whole Zoho books folder the links wouldn't need to be updated if you did it like this, whereas they would if you used the full path, surely this is better if we expect the relative paths of these files never to change?

[the remark this answers is not in the document]
Comment (d-laing) [reply docs/integrations/accounting/zoho-books/accounting-zoho-books.md:21]: Good point. Leave the relative paths for now and we'll discuss as a team.

File: docs/integrations/accounting/zoho-books/zoho-book-limits.md (hunk 2 above)
Comment (jepmatthews) [reply docs/integrations/accounting/zoho-books/zoho-book-limits.md:34]:
```suggestion
Our Zoho Books integration works within these limits to minimize the performance impact of pulling and pushing large volumes of data. To do so, the integration maximises the number of records retrieved per API call for most data types. However, for certain data types, the integration can only fetch a single record for each API call to Zoho Books, impacting performance. This also applies to additional data types if the restrictions on pulling reduced datasets described above are removed.```

Comment (jepmatthews) [reply docs/integrations/accounting/zoho-books/zoho-book-limits.md:34]: Perhaps this is over picky but just wanted to add that it affected the specific data types where you remove restrictions (that's probably obvious but just wanted to make clear), does that wording seem reasonable. Also happy to go with your wording if you think it's clearer

Comment (d-laing) [reply docs/integrations/accounting/zoho-books/zoho-book-limits.md:34]: No, that makes sense - please change. I thought the removal of restrictions by Support applied to all data types (but should have checked with you first).

**Pull request merged**